examples/ginwebapp/main.go

```diff
@@ -126,7 +126,7 @@ func main() {
 	authorized := r.Group("/")
 	authorized.Use(descopegin.AuthneticationMiddleware(client.Auth, nil, nil))
 	authorized.GET("/private", handleIsHealthy)
-	authorized.GET("/logout", handleLogout)
+	authorized.GET("/logout", handleLogout) // Logout from all user's active sessions
```

Comment:
maybe we want to change the logout in our examples to  `POST` as well?

Reply:
done